map_marker_0_yaw: -1.5708
map_marker_0_pitch: 0
map_marker_0_roll: -1.5708
map_marker_0_x: 4
map_marker_0_y: 2
map_marker_0_z: 0.0335
